03_letters/kasunfdo/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func letters(s string) map[rune]int {
+	charMap := map[rune]int{}
+
+	for _, char := range s {
+		charMap[char]++
+	}
+
+	return charMap
+}
+
+func sortLetters(m map[rune]int) []string {
+	out := make([]string, len(m))
```

Comment:
Nit: I like my variables declared as close as possible to there usage (less jumping around when reading) so i'd put `out := ...` just above the second loop.

Reply:
I agree. Thanks Julia! :)